File: thinkup/platform-frontend/src/models/Entity/tech.js
class Tech extends Student {
  /**
   * Inherits Student Class, with the highest position in the role's hierarchy
   * @param {String} name
   * @param {String} email
   * @param {Settings} settings
   * @param {Permissions} perms
   */

  constructor(name, email, settings, perms) {
    super(name);
    super(email);
    super(settings);
    super(perms);
  }
}

File: thinkup/platform-backend/src/model/entity/users/tech.py
from model.entity.users.mentor import Mentor
from model.permissions.permissions import Permissions
from model.settings.settings import Settings


class Tech (Mentor):
    def __init__(self, id: str, name: str, email: str, description: str, settings: Settings, perms: Permissions, project):
        super().__init__(id, name, email, description, settings, perms, project)
